src/ui/components/Dashboard/Menu.jsx

```diff
@@ -14,15 +14,19 @@ import { useQueryClient } from 'react-query';
 function Menu(props) {
   const classes = useStyles();
   const queryClient = useQueryClient();
-  const { open, callback } = props;
+  const { open, setContentKey, callback } = props;
 
+  const setKey = useCallback(() => {
```

Comment:
Since this function always sets the contentKey to config can it be renamed to something more descriptive?